inst/create_lkup.R

```diff
@@ -0,0 +1,7 @@
+
+
+data_dir <- "../pipshpmdp/inst/data/lkup"
+
+lkup_obj <- create_versioned_lkups(data_dir = data_dir)
+
```

Comment:
Why did you add this file under the `inst` folder? Only the code to generate the API should be there.

Reply:
The file is not necessary. I used it to test some things out. I will delete it and push my changes to remove it